### PR TITLE
Pandora029 fluff item

### PR DESCRIPTION
**Pull request by PastelPrinceDan**
https://forum.vore-station.net/viewtopic.php?f=27&t=1986

### CODE REVIEW

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 1]
```diff
@@ -2330,4 +2330,55 @@ Departamental Swimsuits, for general use
...
+/obj/item/clothing/suit/storage/hooded/wintercoat/security/fluff/evelyn
+	name = "warden's navy winter coat"
+	desc = "A custom tailored security winter coat in navy blue colors, this one has the rank markings of a warden on it."
+	icon = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "evelyncoat"
+
+	icon_override = 'icons/vore/custom_onmob_vr.dmi'
+	item_state = "evelyncoat_mob"
+
+	armor = list(melee = 25, bullet = 20, laser = 20, energy = 15, bomb = 20, bio = 0, rad = 0) //same as other sec coats
```
Comment (Novacat) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm:2345]: This is redundant if it is a subtype of security coats. You can safely delete this line.

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 2]
```diff
@@ -2330,4 +2330,55 @@ Departamental Swimsuits, for general use
...
+
+	var/owner = "pandora029"
```
Comment (Novacat) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm:2347]: Delete this, since it will throw an unused var error every time it compiles.

**Review by Novacat — CHANGES_REQUESTED** [review]

**Comment by Novacat** [comment]
Keep in mind that it will inherit the properties of all of its parent objects. Has this been tested?

**Comment by PastelPrinceDan** [comment]
> Keep in mind that it will inherit the properties of all of its parent objects. Has this been tested?

right, removed the two lines. 
i tested it on a private server and everything worked fine yeah

**Review by Novacat — APPROVED** [review]

**Pull request merged**